### PR TITLE
Clean up the readme

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,82 +94,19 @@ attempt to use newer ones, the build will fail. Presently, Rust doesn't allow
 to generate custom error messages so there doesn't appear to be a way to make
 such errors more friendly.
 
-Examples are providing in the [gtk-rs/examples](https://github.com/gtk-rs/examples) repository, you can find some tests showing off the functionality, these can be built and run as follows:
```
Comment (GuillaumeGomez) [inline README.md]: I liked the screenshots. :'(

Comment (gkoz) [inline README.md]: Oh right, I forgot to delete them.
The site can have a "screenshots" section.

Comment (GuillaumeGomez) [inline README.md]: T_T

Comment (gkoz) [inline README.md]: No wait, the screenshots aren't local. So the `resources` directory is an even older artifact.

Comment (GuillaumeGomez) [inline README.md]: I totally forgot about it actually... Remove it as well please.

Comment (gkoz) [inline README.md]: I've moved the installation screenshot there. Till we move all that to the site.

Comment (GuillaumeGomez) [inline README.md]: Seems file by me.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**